webcrypto-shim.js

```diff
@@ -19,7 +19,7 @@ module.exports = function webcryptoShim (global) {
         _SubtleCrypto = global.SubtleCrypto || _subtle.constructor || Object,
         _CryptoKey  = global.CryptoKey || global.Key || Object;
 
-    var isEdge = window.navigator.userAgent.indexOf('Edge/') > -1
+    var isEdge = global.navigator.userAgent.indexOf('Edge/') > -1
```

Comment:
I don't think global is defined in all browsers, maybe check for not begin undefined in this order

1. global
2. window
3. self